docs/download.md

```diff
@@ -87,3 +94,39 @@ Here are the steps to set it up:
 Now that you have everything installed, run `npm run build` inside the Leaflet directory.
 This will combine and compress the Leaflet source files, saving the build to the `dist` folder.
 
+### HTML Overview with Leaflet
+
+If choosing to use the hosted version of Leaflet, the example below can demonstrate how to set up the HTML page.
+```
+index.html
+<html>
+  <head>
+    <title>Leaflet Map</title>
+    
+    <!-- Leaflet CSS -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.8.0/dist/leaflet.css"
```

Comment:
Please use a `latest` link `https://unpkg.com/leaflet@latest/dist/leaflet.css`

Reply:
I'd prefer a specific version — we can use templated one here like above (`{{ site.latest_leaflet_version}}`).

Reply:
In general I would also say a specific version is better but this means maintainance work for us and then I see more benefit if we have less work then have a specific version in the docs.